### PR TITLE
CLOUDP-68636: Add e2e test for org API keys

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-68636

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: e2e/iam/orgs_api_key_test.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+		if err != nil {
+			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		}
+		var keys []mongodbatlas.APIKey
+		if err := json.Unmarshal(resp, &keys); err != nil {
+			t.Fatalf("unexpected error: %v", err)
+		}
+		assert.NotEmpty(t, keys)
+	})
```
Comment (andreaangiolillo) [inline e2e/iam/orgs_api_key_test.go:58]: [ni] I'd run `List` after `Create` to check that we get 1

File: e2e/iam/orgs_test.go [hunk 2]
```diff
@@ -46,28 +39,21 @@ func TestOrgs(t *testing.T) {
...
 
 	t.Run("Describe", func(t *testing.T) {
 		cmd := exec.Command(cliPath, iamEntity, orgEntity, "describe", orgID)
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
-
-		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
-		}
+		assert.NoError(t, err, resp)
```
Comment (andreaangiolillo) [inline e2e/iam/orgs_test.go:57]: 😲

File: e2e/iam/orgs_test.go [hunk 3]
```diff
@@ -46,28 +39,21 @@ func TestOrgs(t *testing.T) {
...
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
+		a := assert.New(t)
+		if a.NoError(err, resp) {
+			var orgs mongodbatlas.Organizations
+			if err := json.Unmarshal(resp, &orgs); err != nil {
+				t.Fatalf("unexpected error: %v", err)
+			}
+			a.NotEmpty(orgs.Results)
+			orgID = orgs.Results[0].ID
 		}
```
Comment (andreaangiolillo) [inline e2e/iam/orgs_test.go:50]: I really like this way of doing it 👏

File: e2e/iam/orgs_api_key_test.go (hunk 1 above)
Comment (gssbzn) [reply e2e/iam/orgs_api_key_test.go:58]: we do, `assert.NotEmpty(t, keys)` already, there's always one, the one we use for the tests ;)

File: e2e/iam/orgs_test.go (hunk 3 above)
Comment (gssbzn) [reply e2e/iam/orgs_test.go:50]: you can thank @robcarlan-mlab for adding `assert` to our libs :)

File: e2e/iam/orgs_api_key_test.go (hunk 1 above)
Comment (andreaangiolillo) [reply e2e/iam/orgs_api_key_test.go:58]: I don't know why I read `assert.Empty`. Thanks

**Pull request merged**